### PR TITLE
Add example use case 01 - form input validator

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/2317

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: examples/form-validator/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[package]
+org = "wso2"
+name = "form_validator"
+version = "0.1.0"
+
+[[dependency]]
+org = "ballerina"
+name = "http"
+version = "2.2.0"
+repository = "central"
```
Comment (daneshk) [inline examples/form-validator/Ballerina.toml:10]:
```suggestion
repository = "central"

```

File: examples/form-validator/form_validator.bal [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    }
+
+    isolated resource function get phone(string value) returns boolean|error {
+        if !validatePhone(value) {
+            return error("Invalid phone number");
+        }
+        return true;
+    }
+
+    isolated resource function post user(@http:Payload User user) returns boolean|string[]|error {
```
Comment (daneshk) [inline examples/form-validator/form_validator.bal:79]:
```suggestion
    isolated resource function post user(@http:Payload User user) returns boolean|error {
```

Comment (daneshk) [reply examples/form-validator/form_validator.bal:79]: we are not returning `string[]` here

Comment (daneshk) [inline examples/form-validator/form_validator.bal:79]: Shall we make it a `get` method?. Post is normally used to create/update resources.

Comment (kaneeldias) [reply examples/form-validator/form_validator.bal]: Fixed in 47d7296

Comment (kaneeldias) [reply examples/form-validator/form_validator.bal]: Fixed in 47d7296

**Review by daneshk — APPROVED** [review]

**Pull request merged**